releng/tools.vitruv.applications.cbs.frameworkwrapper/.project

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>tools.vitruv.applications.cbs.frameworkwrapper</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.ui.externaltools.ExternalToolBuilder</name>
+			<triggers>full,incremental,</triggers>
+			<arguments>
+				<dictionary>
+					<key>LaunchConfigHandle</key>
+					<value>&lt;project&gt;/.externalToolBuilders/org.eclipse.jdt.core.javabuilder.launch</value>
+				</dictionary>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.ui.externaltools.ExternalToolBuilder</name>
+			<triggers>full,incremental,</triggers>
+			<arguments>
+				<dictionary>
+					<key>LaunchConfigHandle</key>
+					<value>&lt;project&gt;/.externalToolBuilders/org.eclipse.xtext.ui.shared.xtextBuilder.launch</value>
+				</dictionary>
+			</arguments>
+		</buildCommand>
```

Comment:
I have to admit that I do not understand what this is necessary for.

Reply:
This is the configuration Eclipse generated after I disabled the Xtext and Java builders for the project. There is no option from the UI to completely remove the builders.

However, I now removed these lines and the references files by hand and Eclipse doesn’t complain. So… I guess we’re good? :shrug:

Reply:
I was only confused, because I did not know what these commands are supposed to do. So it's all fine :-)